tests/test_assignment/gatorgrade.yml

```diff
@@ -0,0 +1,28 @@
+setup: |
+  echo "Setup Complete!"
+---
+- src:
+  - hello-world.py:
+    - description: Complete all TODOs
+      check: MatchFileFragment
+      options:
+        fragment: TODO
+        count: 0
+        exact: true
+    - description: Use an if statement
+      check: MatchFileRegex
+      options:
+        regex: "if .*?:"
+        count: 1
+        exact: false
+  - reflection.md:
+    - description: Complete all TODOs
+      check: MatchFileFragment
+      options:
+        fragment: TODO
+        count: 0
+        exact: true
+- description: Pass MDL
+  command: mdl .
+- description: Pass HTMLHint
+  command: htmlhint
```

Comment:
We should wait until we have support for shell checks to test them in our integration tests.